.changeset/clever-pants-pull.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux/ui5-application-writer': patch
+---
+
+Fix: incorrect settings for ui5-middleware-transpile
```

Comment:
Since other modules are test changes it is fine not to release them

Reply:
I think they will be automatically released as well due to the dependency (even if not needed) I think